.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -8,7 +8,7 @@ body:
       value: |
         Thanks for taking the time to fill out this bug report!
         To expedite issue processing, please search open and closed issues before submitting a new one.
-        Please read our [Code of Conduct](/.github/CODE_OF_CONDUCT.md).
+        Please read our [Code of Conduct](../CODE_OF_CONDUCT.md).
```

Comment:
Sure that it shouldn't be `./.github/CODE_OF_CONDUCT.md`

Reply:
Not 100% sure. If we just take a look at the file system we need to go one level up because of the `ISSUE_TEMPLATES` folder. Unfortunately I did not find an example. I guess we just have to try it out.

Reply:
Is still not working. Seems like `./.github/CODE_OF_CONDUCT.md` should be correct looking at the current path.